test/test_maf.py

```diff
@@ -7,12 +7,12 @@ def test_maf():
     variant_collection_from_maf = load_maf("data/tcga_ov.head.maf")
     expected_variants = [
         Variant(1, 1650797, "A", "G", ensembl),
-        Variant(1, 231401797, "A", "C", ensembl),
         Variant(1, 23836447, "C", "A", ensembl),
+        Variant(1, 231401797, "A", "C", ensembl),
         Variant(11,124617502, "C", "G", ensembl),
     ]
     eq_(len(variant_collection_from_maf), len(expected_variants))
-    for v_expect, v_maf in zip(expected_variants, variant_collection_from_maf):
+    for v_expect, v_maf in zip(expected_variants, sorted(variant_collection_from_maf)):
```

Comment:
Does using `set` inside `VariantCollection` screw up the given order? If so, maybe the `sorted` should go on the `__iter__` method of `VariantCollection`.